js/src/tab.js

```diff
@@ -231,7 +233,6 @@ class Tab extends BaseComponent {
 
     toggle(SELECTOR_DROPDOWN_TOGGLE, CLASS_NAME_ACTIVE)
     toggle(SELECTOR_DROPDOWN_MENU, CLASS_NAME_SHOW)
-    outerElem.setAttribute('aria-expanded', open)
```

Comment:
I suppose this, will break dropdown functionality (either way it has to be removed in next major version)

Reply:
Oups, haven't seen that change, reverting it for sure !